pkgs/development/compilers/ocaml/camlboot.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, enableParallelBuilding ? true
+, guile
+}:
+
+stdenv.mkDerivation rec {
+  name = "camlboot";
```

Comment:
Please use pname + version.

Reply:
(Since there are no official upstream releases, we can call the version `unstable-2021-09-08` for now.)